src/rngs/mod.rs

```diff
@@ -19,26 +19,33 @@
 //! # Background — Random number generators (RNGs)
 //!
 //! Computers are inherently deterministic, so to get *random* numbers one
-//! either has to use a hardware generator or collect small bits of *entropy*
-//! from various sources (e.g. event timestamps, or jitter). Generally the
-//! operating system will collect entropy, remove bias and provide random
-//! values on demand; [`OsRng`] uses this. [`JitterRng`] is a local
-//! implementation of an entropy collector using high-resolution timers. Both
-//! methods have some overhead (the latter especially).
-//!
-//! Usually, therefore, random values are generated using a deterministic
-//! algorithm seeded with random data; a so-called Pseudo-Random Number
-//! Generator (PRNG). The numbers produced are not "true" random numbers but
-//! can be statistically of very high quality and can be impossible to predict.
-//! (They can also have obvious correlations and be trivial to predict; quality
+//! either has to use a hardware generator or collect bits of *entropy* from
+//! various sources (e.g. event timestamps, or jitter). This is a relatively
+//! slow and complicated operation.
+//!
+//! Generally the operating system will collect some entropy, remove bias, and
+//! use that to seed its own PRNG; [`OsRng`] provides an interface to this.
+//! [`JitterRng`] is an entropy collector included with Rand that measures
+//! jitter in the CPU execution time, and jitter in memory access time.
+//! [`EntropyRng`] is a wrapper that uses the best entropy source that is
+//! available.
+//!
+//! ## Pseudo-random number generators
+//!
+//! What is commonly used instead are *pseudo-random number generators* (PRNGs),
```

Comment:
> instead

Stylistically I dislike referring to something in a previous paragraph implicitly, but I'll let it pass.